### ISSUE TITLE
strapi.start is not a function

### ISSUE THREAD

**Issue by Tynyndil** [strapi/strapi#3322]
**Informations**
- **Node.js version**: 11.6.0
- **NPM version**:  6.9.0
- **Strapi version**: 3.0.0-alpha.26.2
- **Database**: MongoDB
- **Operating system**: Windows 10

**What is the current behavior?**

Cannot start my strapi project, i got a "strapi.start is not a function" error. Everything worked fine an hour ago

**Steps to reproduce the problem**

I did a fresh installation on the new alpha26 version recently. Everything worked fine, then i installed a new package and did run an "npm install". Then when i did a strapi start, it shows me  a "cannot find module 'lodash'" error :

![Capture1](https://user-images.githubusercontent.com/20043825/58384488-4ba5bc00-7fe2-11e9-9c1f-0770724d5a70.PNG)

Some research later I found out that the solution could be reinstalling strapi globally, so I did it by running "npm install strapi@alpha -g", and now I have those two issues : 

If i do a new npm install i got this : 

![Capture2](https://user-images.githubusercontent.com/20043825/58384514-b6ef8e00-7fe2-11e9-9258-34db0ed1359b.PNG)

And if a want to start strapi I got this : 
![Capture3](https://user-images.githubusercontent.com/20043825/58384543-1cdc1580-7fe3-11e9-8b44-6e962f356e86.PNG)

**What is the expected behavior?**

Not showing any errors on npm install and starting the strapi server properly. I did make a few modifications and already have a pretty big project so I dont want to start from scratch all over again...

**Suggested solutions**

No idea :/ 

Also thanks a lot for this project, we do love it for our school project at Hetic ;) best regards.

UPDATE : I just remembered I also did an npm cache clean trying to solve that, and adding the post install module didnt change anything

**Comment by bin16** [comment]
Please check your strapi version in package.json. When i deploying my project on server, I found `npm audit fix` would upgrade strapi from `strapi@3.0.0-alpha.26.2` to `strapi@3.0.0-next.8`

``` 
  === npm audit security report ===                        
                                                                                
# Run  npm install strapi@3.0.0-next.8  to resolve 3 vulnerabilities
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Regular Expression Denial of Service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ debug                                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ strapi                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ strapi > koa-router-joi > debug                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/534                             │
└───────────────┴──────────────────────────────────────────────────────────────┘


┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ strapi                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ strapi > koa-ip > lodash                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/782                             │
└───────────────┴──────────────────────────────────────────────────────────────┘


┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ strapi                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ strapi > koa-ip > lodash                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/577                             │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

🤷‍♂️

**Comment by Tynyndil** [comment]
Thanks a lot that was it ! If anybody get's into this issue again just run  "npm install strapi@3.0.0-alpha.26.2 --save" in your project folder to fix the wrong version of your package.json. 

Closing this
